indico/modules/events/editing/schemas.py

```diff
@@ -183,3 +193,9 @@ def _check_for_unique_file_type_name(self, name):
             query = query.filter(EditingFileType.id != file_type.id)
         if query.has_rows():
             raise ValidationError(_('Name must be unique'))
+
+    @validates('extensions')
+    def _check_for_correct_extensions_format(self, extensions):
+        for extension in extensions:
```

Comment:
```python
if any(re.match(r'^[*.]+', ext) for ext in extensions):
    raise ValidationError(_('Extensions cannot have leading dots'))
```

BTW, if this check never happens unless someone messes with the client-side validation, we could skip i18n here.

Reply:
Does it hurt to have it? I would leave it as it is

Reply:
not sure if it was about i18n or replacing the loop with any() or both, but like i said, just a suggestion ;)